src/Role/Repository/RoleHierarchyRepository.php

```diff
@@ -37,9 +37,27 @@ private function getChildIds(array $parentIds): ?array
         $searchCriteria = $this->getDataObjectManager()
             ->whereIn('parent_role_id', $parentIds);
 
+        if($searchCriteria->count() == 0) return [];
+
         return $this->getList($searchCriteria)->get('child_role_id');
     }
 
+    /**
+     * @param array $childIds
+     *
+     * @return array|null
+     */
+    private function getParentIds(array $childIds): ?array
+    {
+        $searchCriteria = $this->getDataObjectManager()
+            ->whereIn('child_role_id', $childIds);
+
+        if($searchCriteria->count() == 0) return [];
```

Comment:
we dont use singleline ifs, please write it out to the fully

if() {
return stuff happens here
}